src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -73,10 +143,10 @@ public TaskInfo createTask(Offer offer, TaskID taskId, NodeTask nodeTask) {
             nmTaskConfig.setAdvertisableCpus(profile.getCpus());
             nmTaskConfig.setAdvertisableMem(profile.getMemory());
             NodeManagerConfiguration nodeManagerConfiguration = this.cfg.getNodeManagerConfiguration();
-            nmTaskConfig.setUser(nodeManagerConfiguration.getUser().orNull());
             nmTaskConfig.setJvmOpts(nodeManagerConfiguration.getJvmOpts().orNull());
             nmTaskConfig.setCgroups(nodeManagerConfiguration.getCgroups().or(Boolean.FALSE));
             nmTaskConfig.setYarnEnvironment(cfg.getYarnEnvironment());
+            nmTaskConfig.setRemoteDistribution(cfg.getMyriadExecutorConfiguration().getNodeManagerUri().isPresent());
```

Comment:
Previously I explicitly placed a boolean remoteDistribution in the conf, we both agreed it was config cruft.  I was anticipating two possibile setups:
1. Hadoop is installed on every node with configs.
2. Hadoop is downloaded on demand and config pulled from the resource manager.
The possibility of the executor binary is remote but the not the hadoop binary, is handled by leaving out the nodeManagerUri and setting the path to hdfs://nn:port/dist/executor.jar.  Having Hadoop installed locally but updating the config via the RM might be useful, but I'd prefer to defer on that as then we're going the need to make some interesting decisions on how that should look.  Likely the easiest would be to download the RM's config, set the HADOOP_CONF_DIR, and the NodeManager would pick up the new config.  This could have some issues though and I'm not currently in a good position to test that idea.